plugins/modules/docker_image_build.py

```diff
@@ -14,10 +14,11 @@
 
 short_description: Build Docker images using Docker buildx
 
-version_added: 3.6.0
+version_added: "3.8.0"
```

Comment:
Please undo the change. The module was added to version 3.6.0 of the collection.